pci/bus_info.rs

```diff
@@ -43,12 +43,23 @@ impl BUS_INFO {
 
     #[cfg(feature = "std")]
     fn get_drm_path(&self, type_name: &str) -> std::io::Result<PathBuf> {
-        let base = PathBuf::from("/dev/dri/by-path");
-
-        let name = format!("pci-{}-{type_name}", self);
-        let link = std::fs::read_link(base.join(name))?;
-
-        std::fs::canonicalize(base.join(link))
+        let base = PathBuf::from("/dev/dri");
+
+        let name = format!("by-path/pci-{}-{type_name}", self);
+        let link = std::fs::read_link(base.join(name));
+        match link {
+            Ok(l) => std::fs::canonicalize(base.join(l)),
```

Comment:
`/dev/dri/by-path/pci-{bus_info}-render` is a link to `../renderD{n}`. So it won't work if base is `/dev/dri`.

Reply:
I'm not sure what you mean. I prefixed the string on line 48 with `by-path/` so that there wouldn't be a problem. Also, I did all this so that I could reuse `base` on line 56. Maybe I'm missing something?

Reply:
The code joins `../renderD{n}` to `base ("/dev/dri")`, so the result is `/dev/dri/../renderD{n} (/dev/renderD{n})`.

https://github.com/Umio-Yasuno/libdrm-amdgpu-sys-rs/pull/4/files#diff-b1792f760d5b13d5042cfc658ba61bad6d0ebdd7f5169ea1232fe8a37adcc173R51

Reply:
Oh, I see where I made the mistake. My bad.